File: Jarvis.Web/MG.Jarvis.Api.Extranet/ViewModel/CancellationPolicyViewModel.cs
using System.Collections.Generic;

namespace MG.Jarvis.Api.Extranet.ViewModel
{
    public class CancellationPolicyViewModel : BaseViewModel
    {
        public int CancellationPolicyId { get; set; }
        public int CancellationPolicyTypeId { get; set; }
        public string CancellationPolicyTypeName { get; set; }
        public bool IsNoShowCharges { get; set; }
        public int? NoShowCancellationChargesId { get; set; }
        public string CancellationChargesName { get; set; }
        public bool IsActive { get; set; }
        public bool IsDeleted { get; set; }
        public string Name { get; set; }
        public CancellationPolicyClausesViewModel CancellationPolicyClausesViewModelList { get; set;}
    }
}

File: Jarvis.Web/MG.Jarvis.Web/ClientApp/src/app/extranet/viewmodel/policiesviewmodel/cancellationpolicyviewmodel.ts
import { ObjectState } from '../../../common/enums';
import { IObjectWithState } from '../../../common/iobjectwithstate';
import { CancellationPolicyClausesViewModel } from './cancellationpolicyclausesviewmodel';

export class CancellationPolicyViewModel implements IObjectWithState {
  cancellationPolicyId: number;
  cancellationPolicyTypeId: number;
  isNoShowCharges: boolean;
  noShowCancellationChargesId: number;
  isActive: boolean;
  isDeleted: boolean;
  name: string;
  hotelId: number;
  cancellationPolicyClausesViewModelList:  CancellationPolicyClausesViewModel;
  objectState: ObjectState;
}
